python/lsst/ip/isr/isrTask.py

```diff
@@ -2261,6 +2286,9 @@ def updateVariance(self, ampExposure, amp, ptcDataset):
         gain = ptcDataset.gain[ampName]
         readNoise = ptcDataset.noise[ampName]
 
+        # The image will always be in adu and the noise
```

Comment:
I think it's important in these comments to say "read noise" (in fact `ptc.noise` was probably a bad name for the variable but we're stuck with it).